src/ConfigEditor/Auth/types.ts

```diff
@@ -7,6 +7,11 @@ export enum AuthMethod {
   CrossSiteCredentials = 'CrossSiteCredentials',
 }
 
+export interface DefaultAuthMethod {
```

Comment:
I'd suggest to rename this type to make it clearer, I think `DefaultAuthMethod` is too broad.

```suggestion
export interface AuthMethodSelectOption {
```